.changesets/fix-mono-dependency-support-for-elixir-1-9.md

```diff
@@ -0,0 +1,5 @@
+---
+bump: "patch"
+---
+
+Fix mono dependency support for Elixir 1.9
```

Comment:
No changeset needed here I think. See my other comment.
This change should not result in a change for the user.